docs/codeql/query-help/codeql-cwe-coverage.rst

```diff
@@ -11,15 +13,22 @@ The CWE categorization contains several types of entity, collectively known as C
 - Weakness Variant
 - Compound Element
 
-Other types of CWE do not correspond directly to weaknesses, so are omitted.
+Other types of CWE that do not correspond directly to weaknesses are omitted.
 
 The CWE categorization includes relationships between entities, in particular a parent-child relationship.
-These relationships are associated with Views (another kind of CWE entity). For the purposes of coverage claims, we use the "[Research View](https://cwe.mitre.org/data/definitions/1000.html)."
+These relationships are associated with Views (another kind of CWE entity). For the purposes of coverage claims, we use the "`Research View <https://cwe.mitre.org/data/definitions/1000.html>`_."
 
 Every security query is associated with one or more CWEs, which are the most precise CWEs that are covered by that query.
 Overall coverage is claimed for the most-precise CWEs, as well as for any of their ancestors in the View.
 
-## Overview
-
-<!-- autogenerated CWE coverage table will be added below -->
-
+.. toctree::
+   :hidden:
+   :titlesonly:
+
+   codeql-cwe-coverage-full
```

Comment:
Since we'll need to update the automation anyway, maybe you could change this article title to something shorter too, to match the others? Something like `all-cwe`?

(definitely not necessary though 😄 )